addons/website/models/res_company.py

```diff
@@ -19,6 +20,15 @@ def action_open_website_theme_selector(self):
         action['target'] = 'new'
         return action
 
+    @api.constrains('active')
+    def _check_active(self):
+        super()._check_active()
+        for company in self:
+            if not company.active and company.website_id:
+                raise ValidationError(_(
+                    'The company %(company_name)s cannot be archived because it has a linked website', company_name=company.name
```

Comment:
It needs to be added inside the `.pot` file.
Maybe worth mentioning which website in the error message, and maybe worth telling "Change that website's company first", no?

Reply:
Do I have to add it manually in the .pot file? It seems to be automated...

Reply:
It's not automated, `.pot` are never touched anymore (except from our manual diff) once a branch has been released.

See https://discord.com/channels/803943906625716244/981917722826903572/1161603743599579207 and https://github.com/odoo/odoo/pull/113570#pullrequestreview-1313445887
It should give you a clear explanation of the `.po`, `.pot` and transifex processes.

Don't hesitate to ask if it needs more clarification